src/internal/_dispatchable.js

```diff
@@ -1,37 +1,22 @@
-var _isArray = require('./_isArray');
 var _isTransformer = require('./_isTransformer');
-var _slice = require('./_slice');
 
 
 /**
- * Returns a function that dispatches with different strategies based on the
- * object in list position (last argument). If it is an array, executes [fn].
- * Otherwise, if it has a function with [methodname], it will execute that
- * function (functor case). Otherwise, if it is a transformer, uses transducer
- * [xf] to return a new transformer (transducer case). Otherwise, it will
- * default to executing [fn].
+ * Returns a function which returns a transformer if its last argument is
+ * a transformer; the result of applying `fn` to its arguments otherwise.
  *
  * @private
- * @param {String} methodname property to check for a custom implementation
  * @param {Function} xf transducer to initialize if object is transformer
  * @param {Function} fn default ramda implementation
  * @return {Function} A function that dispatches on object in list position
  */
-module.exports = function _dispatchable(methodname, xf, fn) {
+module.exports = function _dispatchable(xf, fn) {
```

Comment:
since this is now just for supporting transducers, a name change may be in order, e.g. `_transducible` or `_xducible` or `_xduces` or ...